lib/ssl/test/ssl_session_ticket_SUITE.erl

```diff
@@ -155,6 +161,10 @@ init_per_group(stateful, Config) ->
     [{server_ticket_mode, stateful} | proplists:delete(server_ticket_mode, Config)];
 init_per_group(stateless, Config) ->
     [{server_ticket_mode, stateless} | proplists:delete(server_ticket_mode, Config)];
+init_per_group(stateful_with_cert, Config) ->
+    [{server_ticket_mode, stateful_with_cert} | proplists:delete(server_ticket_mode, Config)];
+init_per_group(stateless_with_cert, Config) ->
+    [{server_ticket_mode, stateless_with_cert} | proplists:delete(server_ticket_mode, Config)];
```

Comment:
If possible, reduce those 4 init_per_group into 1 with guard expression checking for `stateful`, `stateless`, `stateful_with_cert`, `stateless_with_cert`.

Reply:
Fixed